frontend/pages/admin/TeamManagementPage/TeamDetailsWrapper/TeamDetailsWrapper.tsx

```diff
@@ -349,9 +349,9 @@ const TeamDetailsWrapper = ({
     );
   }
 
-  // const hostsCount = currentTeam.host_count;
-  const hostsCount = teams?.length || 1;
-  const hostsTotalDisplay = hostsCount === 1 ? "1 host" : `${hostsCount} hosts`;
+  const hostCount = userTeams.find((thisTeam) => thisTeam.id === currentTeam.id)
+    ?.host_count;
+  const hostsTotalDisplay = hostCount === 1 ? "1 host" : `${hostCount} hosts`;
```

Comment:
It looks like the data we need is returned by the API at `currentTeam.host_count`. However, our `ITeamSummary` interface does not include that property so it's not available. 

Would you please add `host_count` to the `ITeamSummary` interface and reference that? Once that is set you should be able to do `const hostCount = currentTeam.host_count`.

Reply:
yes, I first thought of adding the `host_count` to the `ITeamSummary` interface. But, later decided to follow the other approach